src/components/AddComment.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({comments, review_id}) => {
+    const [commentAuthor, setCommentAuthor] = useState('');
+    const [commentBody, setCommentBody] = useState('');
+    const [commentVotes, setCommentVotes] = useState(0);
+    const [commentPosted, setCommentPosted] = useState(false)
+
+    return <section>
+        <form className="add-comment-form" onSubmit={(event) => {
+            event.preventDefault()
+
+            const commentToAdd = {
+                "author": commentAuthor,
+                "body": commentBody,
+                "votes": commentVotes
+            };
+            setCommentPosted(true)
+            postComment(review_id, commentToAdd)
```

Comment:
You should make sure that users cannot click the submit button multiple times whilst the first POST request is still being processed. Have a think about how you could change your useEffect to use some conditional logic so that postComment only happens if a previous submit hasn't happened yet